content/en/community/hackathons/sessions/baby-steps/work/01-echo-back/main.c

```diff
@@ -16,7 +16,10 @@ int receive_and_send(int client_recvfd, int client_sendfd)
     int bytes_received = 0;
 
     memset(buf, 0, BUFLEN);
+
+	// TODO
```

Comment:
I think there is wrong indentation here. And you can put the message from the bellow line on this one

```suggestion
	// TODO:  recv() buffer into buf;
```